orchagent/srv6orch.cpp

```diff
@@ -423,9 +423,39 @@ bool Srv6Orch::sidEntryEndpointBehavior(string action, sai_my_sid_entry_endpoint
     {
         end_behavior = SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_B6_INSERT_RED;
     }
+    else if (action == "udx6")
```

Comment:
This would be better aligned in a 'switch' case construct.

Reply:
please review if new restructured code is better